examples/version-migration/schema-upgrade/package.json

```diff
@@ -38,13 +38,14 @@
 	"dependencies": {
 		"@fluid-example/example-utils": "workspace:~",
 		"@fluid-experimental/react-inputs": "workspace:~",
+		"@fluid-internal/client-utils": "workspace:~",
 		"@fluidframework/aqueduct": "workspace:~",
 		"@fluidframework/cell": "workspace:~",
-		"@fluidframework/common-utils": "^1.1.1",
 		"@fluidframework/container-definitions": "workspace:~",
 		"@fluidframework/container-loader": "workspace:~",
 		"@fluidframework/container-runtime-definitions": "workspace:~",
 		"@fluidframework/core-interfaces": "workspace:~",
+		"@fluidframework/core-utils": "workspace:~",
```

Comment:
I think I see several instances of packages where the common-utils dep was replaced with both of the new ones, but it only seems to use one or the other (in this case client-utils).

Reply:
Good catch. I found many of these were unused and fixed them in commit 727be7ae298707c466b3b36214f600ce42d6eeb4

To find them, I used this script to remove the deps if present:

```ts
const fs = require("fs");

const pkg = JSON.parse(fs.readFileSync("./package.json"));
if (pkg.dependencies) {
	delete pkg.dependencies["@fluidframework/common-utils"];
	if (
		Object.keys(pkg.dependencies).includes("@fluidframework/core-utils") &&
		Object.keys(pkg.dependencies).includes("@fluid-internal/client-utils")
	)
	delete pkg.dependencies["@fluidframework/core-utils"];
	delete pkg.dependencies["@fluid-internal/client-utils"];
	fs.writeFileSync("./package.json", JSON.stringify(pkg, undefined, "\t"));
} else {
	console.error("No dependency field");
	process.setMaxListeners(1);
}

process.exit(0);
```

I ran it against all packages in the client release group:

```shell
flub exec -g client -- "node /absolute/path/to/script.cjs"
```

Then I ran `pnpm i` and `pnpm build:compile`, fixed the errors, and repeated until done. Finally I merged the changes that survived.